aim/storage/structured/sql_engine/models.py

```diff
@@ -85,3 +93,45 @@ def __init__(self, name):
     def validate_color(self, _, color):
         # TODO: [AT] add color validation
         return color
+
+
+class Note(Base):
+    __tablename__ = 'note'
+    __table_args__ = (
+        UniqueConstraint('run_id', 'name'),
+    )
+
+    id = Column(Integer, autoincrement=True, primary_key=True)
+    name = Column(Text, nullable=False, default='New Note')
```

Comment:
what the `name` field is used for?

Reply:
@alberttorosyan treat is as a file name where you write your notes